samples/sample8/hello_world_cmd_docker.bal

```diff
@@ -6,6 +6,7 @@ import ballerina/docker;
     basePath: "/helloWorld"
 }
 @docker:Config {
+    uberJar: true,
```

Comment:
Do we need to set this `true` ?

Reply:
It is set to true so that the below CMD will work. We can change the CMD or set uberJar to true. WDYT?